processPostcode.ts

```diff
@@ -0,0 +1,21 @@
+import { closest2Stops, getNearbyStops } from "./closestStops";
+import { BusStop, BusStopArrivals, filterBusStop } from "./BusStopHelper";
+import { getPostcodeData, PostcodeInfo } from "./postcodeHelper";
+import { requestBusArrivals } from "./tflRequestHelper";
+import { logger } from "./logHelper";
+
+
+export default async function processPostcode(postcode: string): Promise<BusStopArrivals[]> {

```

Comment:
So lovely and readable! I'd prefer a slightly more descriptive method name - `processPostcode` doesn't really describe what this function does